src/styles/blocks/product.scss

```diff
@@ -0,0 +1,167 @@
+.product {
+  &--catalog {
+    padding-block: 50px;
+    padding-inline: 40px;
+    display: grid;
+    grid-template-rows: 2;
+    grid-template-columns: repeat(1, 200px);
+    row-gap: 46px;
+    column-gap: 48px;
+    justify-content: center;
+
+    @media (min-width: 488px) {
+      grid-template-columns: repeat(2, 200px);
+    }
+
+    @media (min-width: 768px) {
+      grid-template-columns: repeat(3, 200px);
+    }
+
+    @media (min-width: 1024px) {
+      grid-template-columns: repeat(4, 200px);
+    }
+  }
+
+  &__card {
+    padding-left: 16px; /*для сдачи теста*/
+    display: flex;
+    box-sizing: border-box;
+    width: 200px;
+    height: min-content;
+    flex-direction: column;
+    text-align: center;
+    border: 1px solid $whiteColor;
+    border-radius: 5px;
+    transition: 300ms;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+  }
+
+  &__box {
+    width: 166px;
+    margin-inline: auto;
+  }
+
+  &__image {
+    display: block;
+    margin: 32px auto 40px ;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__fonts {
+    margin: 0;
+
+    &--code {
+      margin: 4px 0 0;
```

Comment:
1. reset default margin for `h2`, `p` 
2. try to use either `margin-top` or `margin-bottom` to make vertical indent